egs2/misp2021/asr1/local/enhancement.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+# Copyright 2018 USTC (Authors: Zhaoxu Nian, Hang Chen, Yen-Ju Lu)
+# Apache 2.0
+
+# use nara-wpe and BeamformIt to enhance multichannel data
+
+set -e -o pipefail
+
+# configs
+stage=0
+nj=6
+cmd=run.pl
+
+
```

Comment:
Can you add `.utils/parse_options.sh` here to parse the options from command line